courses/free-courses-hi.md

```diff
@@ -382,6 +382,7 @@
 * [Free Web Development Course For School Students (Grade 5 - 10) \| Certified Course By Coding Blocks Junior \| Learn HTML, CSS, Javascript](https://www.youtube.com/playlist?list=PLhLbJ9UoJCvsCXqP9yAOZpzXHhWkLBXdw) - Coding Blocks Junior
 * [Front End Development Tutorial \| Complete HTML and CSS Tutorial for Beginners (9 Hours)](https://www.youtube.com/watch?v=Eu7G0jV0ImY) - WsCube Tech
 * [HTML CSS & Browser APIs](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC) - Pepcoding
+* [HTML Tutorial for Beginners in Hindi | Complete HTML Course with AI + Notes + 5 Project Bundle](https://youtu.be/k2DSi1zGEc8?si=89wK8McUWsGcJnX3) - CodeWithHarry
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!

Reply:
Thanks for the feedback! The video is structured as a course and is over 2.5 hours long, so, I believe it aligns with the guidelines for single video courses as per the docs. Let me know if there's anything else you'd like me to adjust.

Reply:
I added duration of the video in the file for better clarity.